packages/application/rollup.config.js

```diff
@@ -0,0 +1,35 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import sourcemaps from "rollup-plugin-sourcemaps";
+import postcss from "rollup-plugin-postcss";
```

Comment:
Purely cosmetic and it probably didn't get picked up because this is `.js` instead of `.ts`, but the Lumino convention is for strings to be single quotes, please.